signal_id/utils.py

```diff
@@ -138,3 +138,19 @@ def get_pixel_scales(mywcs):
     #scale = np.array([cdelt[0] * (pc[0,0]**2 + pc[1,0]**2)**0.5,
     # cdelt[1] * (pc[0,1]**2 + pc[1,1]**2)**0.5])
     #return abs(scale[0])
+
+
+def get_celestial_axes(wcs):
```

Comment:
It looks like this isn't being used?  Can any of the functions here: http://astropy.readthedocs.org/en/latest/wcs/#subsetting-and-pixel-scales do what you want?  or http://astropy.readthedocs.org/en/latest/api/astropy.wcs.WCS.html#astropy.wcs.WCS.get_axis_types?